docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -9,7 +9,7 @@ ms.topic: conceptual
 
 # Common NuGet configurations
 
-NuGet's behavior is driven by the accumulated settings in one or more `NuGet.Config` (XML) files that can exist at solution- (project if no solution is used), user-, and computer-wide levels. A global `NuGetDefaults.Config` file also specifically configures package sources. Settings apply to all commands issued in the CLI, the Package Manager Console, and the Package Manager UI.
+NuGet's behavior is driven by the accumulated settings in one or more `NuGet.Config` (XML) files that can exist at solution- (project if no solution is used), user-, and computer-wide levels.
```

Comment:
```suggestion
NuGet's behavior is driven by the accumulated settings in one or more config (XML) files that can exist at solution- (project if no solution is used), user-, and computer-wide levels.
```

There are at least two places where `*.config` files are loaded. The machine wide config location, where VS customers will usually see the "offline" feed config file. And starting from around 2020, maybe 2021, a `config` subdirectory where the user-profile nuget.config file is located.

I expect that the machine wide config directory is documented. If the `config\*.config` pattern isn't also documented, it would be great to add it.

Reply:
Please file an issue for this suggestion, per https://github.com/NuGet/docs.microsoft.com-nuget/pull/3290#issuecomment-2148077312